README.md

```diff
@@ -90,8 +90,8 @@ $ cd ~/gnu/github/exiv2  # location of the project code
 $ mkdir build && cd build
 $ cmake .. -DCMAKE_BUILD_TYPE=Release
 $ cmake --build .
-$ make tests
-$ sudo make install
+$ ctest
+$ sudo cmake --build . --target install
```

Comment:
Or even shorter/recommended `cmake --install .`, but six vs half a dozen really...

Reply:
I'd prefer to keep the `cmake --build . --target install` version because that is consistent with everything else.

I didn't know about `cmake --install .`.  That's getting more involved with cmake than necessary.  It's much the same with conan and ctest, there are many options.  I like to keep life simple and give the user the minimum required for success.